source/_components/remote_rpi_gpio.markdown

```diff
@@ -10,7 +10,8 @@ ha_release: 0.94
 ha_iot_class: Local Push
 ---
 
-The `rpi_gpio` integration is the base for all related GPIO platforms in Home Assistant. There is no setup needed for the integration itself, for the platforms please check their corresponding sections.
+The `rpi_gpio` integration is the base for all related GPIO platforms in Home Assistant. For the platform configurations please check their corresponding sections. The remote RPi(where the GPIO pins you want to control are) and the control computer (where homeassistant is running) must be prepared to run remote_rpi_gpio. The pigpio library must be available on both computers and the pigpiod daemon running on the RPi. See details here: https://gpiozero.readthedocs.io/en/stable/remote_gpio.html. Note that for virtual environments you may need to set an environment variable when starting the environment to set the pin factory, example: Environment =  GPIOZERO_PIN_FACTORY=pigpio PIGPIO_ADDR=YOUR_RPi_IP_ADDRESS. See your operating system documentation for information on how to set this variable.
```

Comment:
This is a lot of wording.. which makes it hard to read.
Try to add some paragraphs, make links actual text links and code fence the environment examples.